src/main/java/org/cyclops/integrateddynamics/core/evaluate/variable/ValueTypeDouble.java

```diff
@@ -125,6 +128,13 @@ public ValueTypeInteger.ValueInteger floor(ValueDouble a) {
         return ValueTypeInteger.ValueInteger.of((int) Math.floor(a.getRawValue()));
     }
 
+    @Override
+    public ValueTypeString.ValueString fuzzy(ValueDouble a) {
+        NumberFormat nf = NumberFormat.getCompactNumberInstance(Locale.US, NumberFormat.Style.SHORT);
```

Comment:
The compacting is very opinionated here.
I suggest moving _all_ number format options to the config file (see `GeneralConfig`), so things like the style, max/min fract digits, max/min int digits.

Reply:
That's a suspiciously convenient configuration system...